content/The_Cartridge_Header.md

```diff
@@ -208,6 +209,8 @@ Specifies the size of the external RAM in the cartridge (if any).
 When using a MBC2 chip $00 must be specified in this entry, even though
 the MBC2 includes a built-in RAM of 512 x 4 bits.
 
+*) No licensed cartridge makes use of this option. Exact behaviour is unknown.
```

Comment:
Actually, where does the "2 KB" number come from? See [this comment](https://github.com/gbdev/pandocs/pull/106#discussion_r567310926).

Reply:
No idea. Maybe an older version the manual, pre-CGB?

I also have a "source" for "MBC4" values: http://www.dotmatrixgame.com/documentation/gb-cartridge-header.html

In general, there seems a lot of noise in the details of the gameboy information. But the 2KB is listed everywhere, so it could have been an early mistake that was copied by everyone. In any case, we cannot just "hide" it from the list now, that would just cause people to wonder why this list is incomplete.

Reply:
Agreed, though I think it'd be better to mention that the source is unknown, then. Or, change the entry to "Unused, *) but unofficial docs list it as 2 KB"?

Reply:
@daid That's not a "source" for MBC4 or anything else. It even plainly states (or at least implies) that the information is copied from the wiki. Looking back through the historical documents in this repo, MBC4 was first mentioned in Martin Korth's 2001 version of Pandocs. I'd say either Martin interpolated the values himself, or find the details in some other doc.

Reply:
I was not serious on the MBC4, it is an example of just bad info out there. That site is pretty much the only location that mentions it now in 2021